partcad/src/partcad/project.py

```
@@ -16,6 +16,7 @@
 import threading
 import typing
 
+from typing import Optional, List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)